packages/webamp/js/components/EqualizerWindow/index.tsx

```diff
@@ -35,6 +37,124 @@ const EqualizerWindow = () => {
     Actions.toggleEqualizerShadeMode
   );
 
+  /**
+   * getBandFromClientX()
+   * Return the band # (60,170,310,etc.) that we are over during the touchmove.
+   * We do this by taking the first band "band-60", and using it's left pos and width
+   *  and calculating the rest of the bands from there.
+   * In this routine we only care about the x axis.
+   * @param clientX 
+   * @returns Band
+   */
+  const getBandFromClientX = (clientX: number) => {
```

Comment:
I don't think this approach (where we bind at the top level window, and then calculate which element the mouse is referring to) is correct. For one it looks very brittle, but for two, I don't think it matches the behavior of Winamp. In Winamp it looks like the you are always dragging the last active slider, not just the one that is closest to your cursor on the x axis:

https://user-images.githubusercontent.com/162735/176981011-0064735a-d25b-4d82-ba50-6c4297c3c2f8.mov

Reply:
Yeah I'm with you. It was quite a hack. I'm tearing all that code out in favor of PointerEvents. In my local dev it's working well and the new code is much more clean. With the video you posted it also shows some other behavior that would need some modification to get in sync with. Like, we don't want to "let go" of the band when you leave the verticalslider area, but only when you release the mouse/touch, or enter another verticalslider. So some changes to how "mouseleave"/"pointerleave" was being utilized.

Reply:
This is fixed and now behaves the same as the video.